gdcdatamodel/models/__init__.py

```diff
@@ -380,7 +379,7 @@ def generate_edge_tablename(src_label, label, dst_label):
         oldname = tablename
         logger.debug('Edge tablename {} too long, shortening'.format(oldname))
         tablename = 'edge_{}_{}'.format(
-            str(hashlib.md5(tablename).hexdigest())[:8],
+            str(hashlib.md5(tablename.encode("latin1")).hexdigest())[:8],
```

Comment:
`uft-8`?

Reply:
md5 hash will differ based on how the string is encoded. `latin1` was default in python2 but looks like we specified encoding in `utf-8` anyway https://github.com/uc-cdis/gdcdatamodel/pull/23/files/f41902a05b8bccd435fd7cc6df7518516a853e3f#diff-1ba9e31512b7e1134d72fe3ea8f7d452L1 so` utf-8` should be fine. How much do we care that these hashes are consistent anyway? If it's super important, we should probably try to come up with some test cases for this.